thamos/cli.py

```diff
@@ -1195,13 +1227,19 @@ def images(
       thamos images --output-format json
 
       thamos images --os-name fedora --os-version 35 --python-version 3.9
+
+      thamos images --symbol GLIBC_FOO
     """
     result = list_thoth_container_images(
         os_name=os_name,
         os_version=os_version,
         python_version=python_version,
         cuda_version=cuda_version,
         image_name=image_name,
+        library_name=library_name,
+        symbol=symbol,
+        package_name=package_name,
+        rpm_package=rpm_package_name,
```

Comment:
I think this will require changes in lib.py so that the function accepts these parameters.